From Pranith-Kumar_D@mentor.com Fri May 18 07:12:54 2007
From: Pranith Kumar D <pranith-kumar_d@mentor.com>
Message-ID: <464DB45C.8070200@mentorg.com>
Date: Fri, 18 May 2007 19:42:44 +0530
To: greg@kroah.com, <penberg@cs.helsinki.fi>, <debiandev@gmail.com>
Subject: HOWTO: removing duplicated entry

This removes the duplicated paragraph in the Documentation/HOWTO file.

Signed-off-by: D Pranith Kumar <pranith-kumar_d@mentorg.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 Documentation/HOWTO |   20 --------------------
 1 file changed, 20 deletions(-)

--- a/Documentation/HOWTO
+++ b/Documentation/HOWTO
@@ -383,26 +383,6 @@ One of the best ways to put into practic
 bugs reported by other people. Not only you will help to make the kernel
 more stable, you'll learn to fix real world problems and you will improve
 your skills, and other developers will be aware of your presence. Fixing
-bugs is one of the best ways to earn merit amongst the developers, because
-not many people like wasting time fixing other people's bugs.
-
-To work in the already reported bug reports, go to http://bugzilla.kernel.org.
-If you want to be advised of the future bug reports, you can subscribe to the
-bugme-new mailing list (only new bug reports are mailed here) or to the
-bugme-janitor mailing list (every change in the bugzilla is mailed here)
-
-	http://lists.osdl.org/mailman/listinfo/bugme-new
-	http://lists.osdl.org/mailman/listinfo/bugme-janitors
-
-
-
-Managing bug reports
---------------------
-
-One of the best ways to put into practice your hacking skills is by fixing
-bugs reported by other people. Not only you will help to make the kernel
-more stable, you'll learn to fix real world problems and you will improve
-your skills, and other developers will be aware of your presence. Fixing
 bugs is one of the best ways to get merits among other developers, because
 not many people like wasting time fixing other people's bugs.
 
